R/backend-access.R

```diff
@@ -22,7 +22,7 @@ sql_select.ACCESS <- function(con, select, from,
     # TOP is expected after DISTINCT and not at the end of the query
     # e.g: SELECT TOP 100 * FROM my_table
     if (!is.null(limit) && !identical(limit, Inf)) {
-      assert_that(is.numeric(limit), length(limit) == 1L, limit > 0)
+      assert_that(is.numeric(limit), length(limit) == 1L, limit >= 0)
```

Comment:
I wonder if we should remove these checks in favour of a single check in `head()`?